s/features/chat/api/cores/persistence/mock-chat-persistence.ts

```diff
@@ -98,10 +98,12 @@ export async function mockChatPersistence(storage: FlexStorage) {
 
 		async addMute(userIds: string[]) {
 			if (userIds.length) {
-				await chatTables.mutes.create(
+				userIds.map(userId => this.isMuted(userId) ? null
```

Comment:
this code will call `chatTables.mutes.create` many times -- instead, we'll prefer to call it only one time (because calling the database is expensive)